### PR TITLE
feat(kiali): kiali plugin

### PR DESCRIPTION
**Pull request by aljesusg**
- Plugins
  -  Kiali
  -  Kiali-Common
  -  Kiali-Backend 

Issue: https://github.com/janus-idp/backstage-plugins/issues/248

# Demo

image

# Issues
- https://github.com/janus-idp/backstage-plugins/issues/552

### CODE REVIEW

File: plugins/kiali-common/src/helpers/config.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { Config } from '@backstage/config';
+
+import { KialiDetails } from '../types';
+
+const KIALI_PREFIX = 'catalog.providers.kiali';
```
Comment (tumido) [inline plugins/kiali-common/src/helpers/config.ts:5]: Please provide a config schema. You can see how we do it in other backend plugins:

https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/config.d.ts

https://backstage.io/docs/conf/defining/

File: plugins/kiali/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@janus-idp/backstage-plugin-kiali",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```
Comment (tumido) [inline plugins/kiali/package.json:7]:
```suggestion
```

**Review by tumido — DISMISSED** [review]
I've just briefly skimmed this for any major flaws and I haven't tested this. So far it looks good. I have found just a few minor issues we need to address. 👍

File: plugins/kiali-backend/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.22.1",
+    "@types/express": "4.17.17",
+    "@types/supertest": "2.0.12",
+    "msw": "1.2.1",
+    "supertest": "6.3.3"
+  },
+  "files": [
+    "dist"
```
Comment (tumido) [inline plugins/kiali-backend/package.json:49]: please add `config.d.ts` to the dist here and also as:

```json
  "configSchema": "config.d.ts",
```

**Review by tumido — CHANGES_REQUESTED** [review]

**Review by aljesusg — COMMENTED** [review]
Changes done

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by aljesusg** [comment]
> /lgtm
> 
> /approve

Hooray!

**Comment by jasperchui** [comment]
resolves #552

**Comment by aljesusg** [comment]
> resolves #552

Sorry, infact this resolves https://github.com/janus-idp/backstage-plugins/issues/248

**Comment by aljesusg** [comment]
Sorry bu we are new in janus. Do we need to do anything else to merge this?

**Comment by tumido** [comment]
No need to do anything. We had release tooling issues. It should be resolved now. 🙂 Merging. 👍

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Comment by aljesusg** [comment]
Great thanks @tumido We did the documentation PR https://github.com/janus-idp/janus-idp.github.io/pull/183 !!!

**Pull request merged**